### PR TITLE
[4.0] com_categories [a11y] tooltips

### PR DESCRIPTION
**Pull request by brianteeman**
Replace title with accessible tooltip

PR for #34602

demo

### CODE REVIEW

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on d3bd8d7967660d03623bc6349502ae9aaeb8aec2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34615">issues.joomla.org/tracker/joomla-cms/34615</a>.</sub>

**Comment by ricardo1709** [comment]
I have tested this item :white_check_mark: successfully on d3bd8d7967660d03623bc6349502ae9aaeb8aec2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34615">issues.joomla.org/tracker/joomla-cms/34615</a>.</sub>

**Comment by chmst** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34615">issues.joomla.org/tracker/joomla-cms/34615</a>.</sub>

File: administrator/components/com_categories/tmpl/categories/default.php [hunk 1]
```diff
@@ -209,26 +209,50 @@
...
 									<?php if (isset($this->items[0]) && property_exists($this->items[0], 'count_published')) : ?>
 										<td class="text-center btns d-none d-md-table-cell itemnumber">
-											<a class="btn <?php echo ($item->count_published > 0) ? 'btn-success' : 'btn-secondary'; ?>" title="<?php echo Text::_('COM_CATEGORY_COUNT_PUBLISHED_ITEMS'); ?>" href="<?php echo Route::_('index.php?option=' . $component . ($section ? '&view=' . $section : '') . '&filter[category_id]=' . (int) $item->id . '&filter[published]=1&filter[level]=1'); ?>">
-												<?php echo $item->count_published; ?></a>
+											<a class="btn <?php echo ($item->count_published > 0) ? 'btn-success' : 'btn-secondary'; ?>"
+												href="<?php echo Route::_('index.php?option=' . $component . ($section ? '&view=' . $section : '') . '&filter[category_id]=' . (int) $item->id . '&filter[published]=1&filter[level]=1'); ?>"
+												aria-describedby="tip<?php echo $i; ?>">
+												<?php echo $item->count_published; ?>
+											</a>
+											<div role="tooltip" id="tip<?php echo $i; ?>">
```
Comment (Quy) [inline administrator/components/com_categories/tmpl/categories/default.php]: `id` should be unique.

Comment (brianteeman) [reply administrator/components/com_categories/tmpl/categories/default.php]: oops - updating now

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 1384deb1cec02bc01aaf6542da57b4f5af94bc57<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34615">issues.joomla.org/tracker/joomla-cms/34615</a>.</sub>

File: administrator/components/com_categories/tmpl/categories/default.php [hunk 2]
```diff
@@ -209,26 +209,50 @@
...
+											</div>
 										</td>
 									<?php endif; ?>
 									<?php if (isset($this->items[0]) && property_exists($this->items[0], 'count_archived')) : ?>
 										<td class="text-center btns d-none d-md-table-cell itemnumber">
-											<a class="btn <?php echo ($item->count_archived > 0) ? 'btn-info' : 'btn-secondary'; ?>" title="<?php echo Text::_('COM_CATEGORY_COUNT_ARCHIVED_ITEMS'); ?>" href="<?php echo Route::_('index.php?option=' . $component . ($section ? '&view=' . $section : '') . '&filter[category_id]=' . (int) $item->id . '&filter[published]=2&filter[level]=1'); ?>">
-												<?php echo $item->count_archived; ?></a>
+											<a class="btn <?php echo ($item->count_archived > 0) ? 'btn-info' : 'btn-secondary'; ?>"
+											href="<?php echo Route::_('index.php?option=' . $component . ($section ? '&view=' . $section : '') . '&filter[category_id]=' . (int) $item->id . '&filter[published]=2&filter[level]=1'); ?>"
+											aria-describedby="tip-archive<?php echo $i; ?>">
```
Comment (Quy) [inline administrator/components/com_categories/tmpl/categories/default.php]:
```suggestion
												href="<?php echo Route::_('index.php?option=' . $component . ($section ? '&view=' . $section : '') . '&filter[category_id]=' . (int) $item->id . '&filter[published]=2&filter[level]=1'); ?>"
												aria-describedby="tip-archive<?php echo $i; ?>">
```

File: administrator/components/com_categories/tmpl/categories/default.php [hunk 3]
```diff
@@ -209,26 +209,50 @@
...
+											</div>
 										</td>
 									<?php endif; ?>
 									<?php if (isset($this->items[0]) && property_exists($this->items[0], 'count_trashed')) : ?>
 										<td class="text-center btns d-none d-md-table-cell itemnumber">
-											<a class="btn <?php echo ($item->count_trashed > 0) ? 'btn-dark' : 'btn-secondary'; ?>" title="<?php echo Text::_('COM_CATEGORY_COUNT_TRASHED_ITEMS'); ?>" href="<?php echo Route::_('index.php?option=' . $component . ($section ? '&view=' . $section : '') . '&filter[category_id]=' . (int) $item->id . '&filter[published]=-2&filter[level]=1'); ?>">
-												<?php echo $item->count_trashed; ?></a>
+											<a class="btn <?php echo ($item->count_trashed > 0) ? 'btn-dark' : 'btn-secondary'; ?>"
+											href="<?php echo Route::_('index.php?option=' . $component . ($section ? '&view=' . $section : '') . '&filter[category_id]=' . (int) $item->id . '&filter[published]=-2&filter[level]=1'); ?>"
+											aria-describedby="tip-trash<?php echo $i; ?>">
```
Comment (Quy) [inline administrator/components/com_categories/tmpl/categories/default.php]:
```suggestion
												href="<?php echo Route::_('index.php?option=' . $component . ($section ? '&view=' . $section : '') . '&filter[category_id]=' . (int) $item->id . '&filter[published]=-2&filter[level]=1'); ?>"
												aria-describedby="tip-trash<?php echo $i; ?>">
```

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**